servus/serializable.h

```diff
@@ -99,23 +100,28 @@ class Serializable
 #endif
 
     /**
-     * Set a new function called after the object has been updated.
-     *
-     * @return the previously set function.
+     * Set a new function called after every change on the local object
+     * instance (e.g. Collage distributable).
      */
-    SERVUS_API ChangeFunc setUpdatedFunction( const ChangeFunc& func );
+    SERVUS_API void setLocalChangeFunction( const ChangeFunc& func );
+
+    /**
+     * Set a new function called after the object has been updated remotely
+     * (via a subscriber, a http server, loading from file...).
+     */
+    SERVUS_API void setRemoteUpdateFunction( const ChangeFunc& func );
 
     /**
      * Set a new function called when a request has been received.
-     *
      * Invoked before the object is published.
-     * @return the previously set function.
      */
-    SERVUS_API ChangeFunc setRequestedFunction( const ChangeFunc& func );
+    SERVUS_API void setRequestedFunction( const ChangeFunc& func );
 
-    /** @internal used by ZeroEQ to invoke updated function */
-    SERVUS_API void notifyUpdated() const;
-    /** @internal used by ZeroEQ to invoke updated function */
+    /** @internal used by ZeroEQ to invoke 'localChange' function */
+    SERVUS_API void notifyLocalChange() const;
+    /** @internal used by ZeroEQ to invoke 'remoteUpdate' function */
```

Comment:
Very good, I was also suggesting in the other(?) PR that notifyRemoteUpdate() and notifyRequested() can be made private and called internally by the public to/fromJSON/Binary().